content/hi/docs/concepts/architecture/nodes.md

```diff
@@ -0,0 +1,649 @@
+नाम एक ही वस्तु है। नोड के मामले में, यह स्पष्ट रूप से माना जाता है कि एक उदाहरण का उपयोग कर रहा है
+समान नाम की स्थिति समान होगी (उदा. नेटवर्क सेटिंग्स, रूट डिस्क सामग्री)
+और विशेषताएँ जैसे नोड लेबल। इससे हो सकता है
+विसंगतियां अगर किसी उदाहरण को उसका नाम बदले बिना संशोधित किया गया था। यदि नोड की आवश्यकता है
+प्रतिस्थापित या महत्वपूर्ण रूप से अपडेट किया गया, मौजूदा नोड ऑब्जेक्ट को एपीआई सर्वर से हटाने की जरूरत है
+अपडेट के बाद पहले और फिर से जोड़ा गया।
+
+### नोड्स का स्व-पंजीकरण
+
+जब क्यूबलेट फ्लैग `--register-node` ट्रू (डिफ़ॉल्ट) होता है, तो क्यूबलेट प्रयास करेगा
+एपीआई सर्वर के साथ खुद को पंजीकृत करें। यह पसंदीदा पैटर्न है, जिसका उपयोग अधिकांश डिस्ट्रोज़ द्वारा किया जाता है।
+
+स्व-पंजीकरण के लिए, क्यूबलेट निम्नलिखित विकल्पों के साथ शुरू होता है:
+
+- `--kubeconfig` - API सर्वर पर स्वयं को प्रमाणित करने के लिए क्रेडेंशियल्स का पथ।
+- `--cloud-provider` - {{< glosary_tooltip text="क्लाउड प्रोवाइडर" टर्म_आईडी="क्लाउड-प्रोवाइडर" >}} से कैसे बात करें अपने बारे में मेटाडेटा पढ़ने के लिए।
+- `--register-node` - स्वचालित रूप से एपीआई सर्वर के साथ पंजीकरण करें।
+- `--register-with-taints` - दी गई सूची के साथ नोड को पंजीकृत करें
+  {{< glosary_tooltip text="taints" term_id="taint" >}} (अल्पविराम से अलग `<कुंजी>=<मान>:<प्रभाव>`)।
+  नो-ऑप अगर `रजिस्टर-नोड` गलत है।
+- `--node-ip` - नोड का आईपी पता।
+- `--node-labels` - {{< glossary_tooltip text="Labels" term_id="label" >}} नोड को पंजीकृत करते समय जोड़ने के लिए क्लस्टर में (द्वारा लागू लेबल प्रतिबंध देखें
+  [NodeRestriction प्रवेश प्लगइन](/docs/reference/access-authn-authz/admission-controllers/#noderestriction))।
+- `--node-status-update-frequency` - निर्दिष्ट करता है कि कितनी बार क्यूबलेट अपनी नोड स्थिति को API सर्वर पर पोस्ट करता है।
+
+जब [नोड प्राधिकरण मोड](/docs/reference/access-authn-authz/node/) और
+[NodeRestriction प्रवेश प्लगइन](/docs/reference/access-authn-authz/admission-controllers/#noderestriction)
+सक्षम हैं, क्यूबलेट केवल अपने स्वयं के नोड संसाधन बनाने/संशोधित करने के लिए अधिकृत हैं।
+
+{{< नोट >}}
+[Node name Uniqueness](#node-name-uniqueness) सेक्शन में बताया गया है,
+जब नोड कॉन्फ़िगरेशन को अद्यतन करने की आवश्यकता होती है, तो पुनः पंजीकरण करना एक अच्छा अभ्यास है
+एपीआई सर्वर के साथ नोड। उदाहरण के लिए, यदि क्यूबलेट को फिर से शुरू किया जा रहा है
+`--नोड-लेबल` का नया सेट, लेकिन उसी नोड नाम का उपयोग किया जाता है, परिवर्तन होगा
+प्रभावी नहीं होगा, क्योंकि नोड पंजीकरण पर लेबल सेट किए जा रहे हैं।
+
+नोड पर पहले से ही निर्धारित पॉड्स गलत व्यवहार कर सकते हैं या समस्याएँ पैदा कर सकते हैं यदि नोड
+क्यूबलेट पुनरारंभ पर कॉन्फ़िगरेशन बदल जाएगा। उदाहरण के लिए, पहले से चल रहा है
+पॉड को नोड को सौंपे गए नए लेबल के खिलाफ दागी जा सकता है, जबकि अन्य
+पॉड्स, जो उस पॉड के साथ असंगत हैं, इस नए के आधार पर शेड्यूल किए जाएंगे
+लेबल। नोड का पुनः पंजीकरण सुनिश्चित करता है कि सभी पॉड्स खाली और ठीक से हो जाएंगे
+पुनर्निर्धारित।
+{{< /नोट >}}
+
+### मैनुअल नोड प्रशासन
+
+आप नोड ऑब्जेक्ट्स का उपयोग करके बना और संशोधित कर सकते हैं
+{{< glosary_tooltip text="kubectl" term_id="kubectl" >}}.
```

Comment:
Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.

```suggestion
{{< glossary_tooltip text="क्यूबकण्ट्रोल" term_id="kubectl" >}}.
```